src/components/Breadcrumbs/index.tsx

```diff
@@ -16,7 +16,11 @@ type Props = {
 };
 
 const overrides = {
-  '/': 'Home'
+  '/': 'Home',
+  '/javascript/prev': 'V6 (Preview)',
```

Comment:
These are overrides for the framework versions in the breadcrumbs so that each one will say what it currently says on prod.